Findologic/Search/Controller/Export/ExportController.php

```diff
@@ -3,7 +3,7 @@
 namespace Findologic\Search\Controller\Export;
 
 use FINDOLOGIC\Export\Exporter;
-use Findologic\Search\Example\XmlExample;
+use Findologic\MagentoExport\Export;
```

Comment:
Why is it called MagentoExport?

Reply:
To be clear it is for magento and it is the export.

Reply:
But where are here at `plugin-magento-2-di` and the plugin runs in Magento, so this should be clear.
Can we just go with `Findologic\Export'?

Reply:
> Can we just go with Findologic\Export?

Well.. what if I told you that `libflexport` already uses that namespace:

![image](https://user-images.githubusercontent.com/20555112/92609965-74c18d00-f2b7-11ea-8051-2f85a3321718.png)

Reply:
Ok leave it as it is :)